README.md

```
@@ -24,164 +24,94 @@ Website 🚀 <a href="https://contributte.org">contributte.org</a> | Contact 
-    1. Setup PostgreSQL 10. You can start it manually or use docker image `dockette/postgres:15`.
+```bash
+composer create-project -s dev contributte/apitte-skeleton
+cd apitte-skeleton
+cp config/local.neon.dist config/local.neon  # adjust DB credentials if needed
+docker-compose up
+```
 
-       ```bash
-       docker run -it -p 5432:5432 -e POSTGRES_PASSWORD=contributte -e POSTGRES_USER=contributte dockette/postgres:15
-       ```
+Once the containers are up, open http://localhost:8000 and browse the sample endpoints:
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)